docs/tutorial/part-four/index.md

```diff
@@ -1,106 +1,87 @@
 
-GraphQL is a **q**uery **l**anguage (the _QL_ part of its name). If you're
-familiar with SQL, it works in a very similar way. Using a special syntax, you describe
-the data you want in your component and then that data is given
-to you.
+GraphQL a été inventé chez Facebook pour aider les ingénieurs à _récupérer_ les données dans des composants
 
-Gatsby uses GraphQL to enable components to declare the data they need.
+GraphQL est un **l**anguage de **q**uery (en anglais **q**uery **l**anguage, la partie _QL_ de son nom). Si vous êtes familiers avec SQL, il fonctionne d'une façon similaire. Utilisant une syntaxe spéciale, vous décrivez la donnée que vous voulez dans votre component et la donnée vous est ensuite transmise.
 
-## Create a new example site
+Gatsby utilise GraphQL pour permettre aux composants de déclarer les données dont ils ont besoin.
 
-Create another new site for this part of the tutorial. You're going to build a Markdown blog called "Pandas Eating Lots". It's dedicated to showing off the best pictures and videos of pandas eating lots of food. Along the way, you'll be dipping your toes into GraphQL and Gatsby's Markdown support.
+## Créer un nouveau site exemple
 
-Open a new terminal window and run the following commands to create a new Gatsby site in a directory called `tutorial-part-four`. Then navigate to the new directory:
+Créez un nouveau site pour cette partie du tutoriel. Vous allez créer un blog Markdown appelé "Les Pandas Mangent Beaucoup". Il est dédié à montrer les meilleures images et vidéos de pandas mangeant beaucoup de nourriture. En cours de route, vous plongerez dans les bases de GraphQL et du Markdown de Gatsby.
+
+Ouvrez un nouveau terminal et lancez les commandes suivantes pour créer un site Gatsby dans un répertoire appelé `tutorial-part-four`. Ensuite naviguez dans le nouveau dossier :
 
 ```shell
 gatsby new tutorial-part-four https://github.com/gatsbyjs/gatsby-starter-hello-world
 cd tutorial-part-four
 ```
 
-Then install some other needed dependencies at the root of the project. You'll use the Typography theme
-"Kirkham", and you'll try out a CSS-in-JS library, ["Emotion"](https://emotion.sh/):
+Ensuite, installez d'autres dépendances nécessaires à la racine du projet. Vous allez utiliser le thème de la typographie "Kirkham", et vous allez essayer une librairie CSS-in-JS, ["Emotion"](https://emotion.sh/):
 
 ```shell
 npm install --save gatsby-plugin-typography typography react-typography typography-theme-kirkham gatsby-plugin-emotion @emotion/core
 ```
 
-Set up a site similar to what you ended with in [Part Three](/tutorial/part-three). This site will have a layout component and two page components:
+Mettre en place un site similaire à ce que vous avez fini avec en [Partie Trois](/tutorial/part-three). Ce site aura un composant de mise en page et deux composants de page :
```

Comment:
```suggestion
Mettre en place un site similaire à ce que vous avez réalisé dans la [partie Trois](/tutorial/part-three). Ce site aura un composant de mise en page et deux composants de page :
```

Reply:
@dantereve Why "partie Trois". We should choose between "partie trois" or "Partie Trois".

I personally prefer "Partie Trois" as it is kinda like a title.

Reply:
Why "partie Trois ?" simply because I've missed the shift key on my new keyboard 🤣. It looks like a title to me too. But doing so will create some inconsistency towards the other links in this part of the tutorial but also every part. I think we put those kinds of links all lowercase, right? We can change that globally though. What do you think?

Reply:
After considering it, it is better to have the title in uppercase for each first letter. 👍